ethportal-api/src/dashboard/grafana.rs

```diff
@@ -12,6 +12,8 @@ pub struct GrafanaAPI {
     address: String,
 }
 
+// todo: automatically update datasource/dashboard via `create-dashboard` command
+// rather than deleting and recreating them
```

Comment:
To issue?

Reply:
Well... Imo, this is a very low priority issue. Given the increased activity of our stale issue bot, I don't think it will be addressed anytime soon and will simply be marked as stale before anybody fixes this. 

Given that I updated the docs (in `book/src/users/monitoring.md`) I just went ahead and removed this `todo`. But lmk if you think it's worth an issue & I'll create one

Reply:
Eh, I guess I'm still not fully sold on the stale bot. The idea that we are hesitant to create issues that aren't going to be quickly resolved feels like an anti-pattern to me. I don't really see why TODOs peppered throughout the code is any better than a backlog of issues that have been around for a while (I'd argue it is worse, in fact. Partially because it's hard to have a conversation in a code TODO). But I guess the solution I'm leaning toward would handle the TODOs in code: https://github.com/ethereum/trin/issues/932 so I'm fine leaving it for now